code/game/machinery/botlaunchpad.dm

```diff
@@ -0,0 +1,85 @@
+/obj/machinery/botpad
+	name = "Bot pad"
+	desc = "A lighter version of the orbital mech pad modified to launch bots. Requires linking to a remote to function."
+	icon = 'icons/obj/telescience.dmi'
+	icon_state = "botpad"
+	circuit = /obj/item/circuitboard/machine/botpad
+	// ID of the console, used for linking up
+	var/id = "botlauncher"
+	var/obj/item/botpad_remote/connected_remote
+	var/mob/living/simple_animal/bot/launched_bot // we need this to recall the bot
+
+/obj/machinery/botpad/Initialize(mapload)
+	. = ..()
```

Comment:
Isn't this just implied? If you don't override Initialize then the parent is the only thing that runs.